packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.swift

```diff
@@ -27,6 +27,21 @@ public class InAppPurchasePlugin: NSObject, FlutterPlugin, InAppPurchaseAPI {
   // This property is optional, as it requires self to exist to be initialized.
   public var paymentQueueHandler: FLTPaymentQueueHandlerProtocol?
 
+  // This should be an Task, but Task is on available >= iOS 13
+  private var _updateListenerTask: Any?
+
+  @available(iOS 13.0, *)
+  var getListenerTaskAsTask: Task<(), Never> {
```

Comment:
this should be called `var updateListenerTask: Task<(), Never>` (no `get` in getters naming convention)